src/Game.js

```diff
@@ -1,9 +1,68 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Container from '@material-ui/core/Container';
+import TextField from '@material-ui/core/TextField';
+import { connect } from 'react-redux';
+
+const paragraphs = [
```

Comment:
you can keep this here, but sometimes i move stuff like this to a constants file (Game.constants.js perhaps) and import it here. 

mostly because large blocks of text distract me from the component focused parts of this app.